SPECS/kernel-hci/kernel-hci.spec

```diff
@@ -55,6 +55,250 @@ Patch24:        0025-net-mlx5-Bridge-implement-infrastructure-for-VLAN-pr.patch
 Patch25:        0026-net-mlx5-Bridge-implement-QinQ-support.patch
 Patch26:        0027-mstflint-This-driver-enables-under-the-secure-boot.patch
 Patch27:        0028-net-mlx5-Bridge-use-debug-not-warn-if-entry-not-found.patch
+Patch28:	0029-ice-Add-DSCP-support.patch                                
```

Comment:
Should we club all these patches in a single file.

Maybe something like this:
```bash
#bash
for i in {0028..0272}; do
     cat $i*.patch >> support-intel-e800.patch
     git rm $i*.patch
done
```

Reply:
I can, but was advised by ChaitanyaL to do them as individual patches.  If you prefer them all as one patch.  I will update.

Reply:
Will base off Main and submit new PR from that branch and will try single patch approach.

Reply:
If someone had advised to do in a specific way, let's prefer that.